### PR TITLE
fix: placeholder's position fixed in rtl

### LINKED ISSUE #2041: Select placeholder in RTL is wrongly postioned
#### Is this a bug, enhancement, or feature request?
bug
#### Briefly describe your proposal.
Screen Shot 2020-02-20 at 6 35 12 PM
Screen Shot 2020-02-20 at 6 34 54 PM

### PR DESCRIPTION
**Pull request by rengare**
#### Please provide a link to the associated issue.
Fixes #2041
#### Please provide a brief summary of this pull request.

Rtl service added in select component to handle rtl

### Before:
image

### After:
image

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done

### CODE REVIEW

File: libs/platform/src/lib/components/select/select.component.ts [hunk 1]
```diff
@@ -7,9 +7,10 @@ import {
     ChangeDetectorRef,
     ContentChild,
     TemplateRef,
-    AfterContentInit } from '@angular/core';
+    AfterContentInit
+} from '@angular/core';
 import { NG_VALUE_ACCESSOR } from '@angular/forms';
-import { OptionComponent, SelectComponent as fdSelect } from '@fundamental-ngx/core';
+import { OptionComponent, SelectComponent as fdSelect, RtlService } from '@fundamental-ngx/core';
```
Comment (mikerodonnell89) [inline libs/platform/src/lib/components/select/select.component.ts]: Importing these from the lib causes a circular dependency

File: libs/core/src/lib/select/select.component.ts [hunk 2]
```diff
@@ -163,7 +168,7 @@ export class SelectComponent implements OnChanges, AfterContentInit, OnInit, OnD
     /** @hidden */
     onTouched: Function = () => { };
 
-    constructor(private changeDetectorRef: ChangeDetectorRef) { }
+    constructor(private changeDetectorRef: ChangeDetectorRef, @Optional() private rtlService: RtlService) { }
```
Comment (salarenko) [inline libs/core/src/lib/select/select.component.ts]: `_` prefix

File: libs/core/src/lib/select/select.component.ts [hunk 3]
```diff
@@ -185,7 +190,12 @@ export class SelectComponent implements OnChanges, AfterContentInit, OnInit, OnD
     }
 
     ngOnInit() {
-        // console.log(this.loading);
+        if (this.rtlService) {
+            this.rtlService.rtl
```
Comment (salarenko) [inline libs/core/src/lib/select/select.component.ts]: Unsubscribe missing and can be simplified to observable

File: libs/core/src/lib/select/select.component.ts (hunk 2 above)
Comment (rengare) [reply libs/core/src/lib/select/select.component.ts]: prefix added

File: libs/core/src/lib/select/select.component.ts (hunk 3 above)
Comment (rengare) [reply libs/core/src/lib/select/select.component.ts]: observable used instead

**Pull request merged**